### PR TITLE
Add infobox for view

### PR DESCRIPTION
**Pull request by pickfire**

### CODE REVIEW

File: helix-term/src/commands.rs [hunk 1]
```diff
@@ -3384,47 +3385,41 @@ fn select_register(cx: &mut Context) {
...
-            }
-        }
-    })
+fn align_view_bottom(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+    align_view(doc, view, Align::Bottom);
+}
+
+// TODO: fix naming
+fn view_what(cx: &mut Context) {
```
Comment (archseer) [inline helix-term/src/commands.rs:3404]: This is middle as in the middle of the line (horizontal centering)

Comment (pickfire) [reply helix-term/src/commands.rs:3404]: But it didn't go to the middle for me.

File: helix-term/src/commands.rs [hunk 2]
```diff
@@ -3384,47 +3385,40 @@ fn select_register(cx: &mut Context) {
...
+    let pos = coords_at_pos(doc.text().slice(..), pos);
+
+    const OFFSET: usize = 7; // gutters
+    view.first_col = pos
+        .col
+        .saturating_sub(((view.area.width as usize).saturating_sub(OFFSET)) / 2);
+}
+
+fn scroll_up(cx: &mut Context) {
+    let count = cx.count.unwrap_or_else(|| NonZeroUsize::new(1).unwrap());
```
Comment (archseer) [inline helix-term/src/commands.rs:3415]: `let count = cx.count()`

File: helix-term/src/commands.rs [hunk 3]
```diff
@@ -3384,47 +3385,40 @@ fn select_register(cx: &mut Context) {
...
+    scroll(cx, count.get(), Direction::Backward);
+}
+
+fn scroll_down(cx: &mut Context) {
+    let count = cx.count.unwrap_or_else(|| NonZeroUsize::new(1).unwrap());
```
Comment (archseer) [inline helix-term/src/commands.rs:3420]: `let count = cx.count()`

**Review by archseer — CHANGES_REQUESTED** [review]

**Pull request merged**